setup.py

```diff
@@ -13,6 +13,10 @@
     'six',
 ]
 
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), 'rb') as f:
+    long_description = f.read().decode('utf-8-sig')
```

Comment:
any reason for the `utf-8-sig` decoding here, rather than opening the file in `utf8`?

Reply:
`README.md` is a text file in UTF-8 encoding. There is no common agreement about byte order mark presence in markdown files. Some build pipelines require it, some require markdown to not have BOM. `utf-8-sig` codec consumes byte order mark if it is present in stream and returns string untouched if it is not present.

In short, `-sig` version of `utf-8` codec allows setup script to be BOM-agnostic reading `README.md`.

I've checked this script with BOM-marked and not marked version of same file and hashes of retrieved unicode string were equal.